.github/workflows/loom.yml

```diff
@@ -95,3 +95,19 @@ jobs:
         working-directory: tokio
         env:
           SCOPE: ${{ matrix.scope }}
+
+  loom-util-sync:
+    name: loom tokio-util::sync
+    # base_ref is null when it's not a pull request
+    if: github.repository_owner == 'tokio-rs' && (contains(github.event.pull_request.labels.*.name, 'R-loom-util-sync') || (github.base_ref == null))
```

Comment:
You could convert this PR as draft, and then temporarily enable this during the development.

Reply:
Fair enough, doing that